src/core/js/device.js

```diff
@@ -110,11 +110,11 @@ define([
 
     if (os === '') {
       // Fall back to using navigator.platform in case Bowser can't detect the OS.
-      var platform = navigator.platform;
-      var platforms = ['Win', 'Mac', 'Linux'];
+      const platform = navigator.platform;
+      const platforms = ['Win', 'Mac', 'Linux'];
       os = 'PlatformUnknown';
 
-      for (var j = 0; j < platforms.length; j++) {
+      for (let j = 0; j < platforms.length; j++) {
```

Comment:
Bump.

Reply:
Its more this below innit?
```js
os =  (platforms.find(name => platform.includes(name)) || '').toLowerCase() || os;`
```
Platform will be a longer browser string which should contain one of the platforms sub strings to match?

Reply:
```js
      // Fall back to using navigator.platform in case Bowser can't detect the OS.
      const platform = navigator.platform.toLowerCase();
      const match = platform.match(/win|mac|linux/);
      if (match) os = match[0];
      // Set consistency with the Bowser flags.
      if (os === 'win') os = 'windows';
      if (!os) os = 'PlatformUnknown';
```